### PR TITLE
General backgrounds and borders design token updates

### PR DESCRIPTION
**Pull request by GuillaumeRx**
Fixes: #13719

Explanation:  Replace backgrounds and borders with design tokens colors

Manual testing steps:  
  - 
  - 
  -

### CODE REVIEW

File: ui/pages/routes/routes.component.js [hunk 1]
```diff
@@ -117,6 +117,8 @@ export default class Routes extends Component {
         pageChanged(locationObj.pathname);
       }
     });
+
+    document.documentElement.setAttribute('data-theme', 'dark');
```
Comment (darkwing) [inline ui/pages/routes/routes.component.js:121]: Note: we need to remove this hardcoded theme code.

Comment (darkwing) [reply ui/pages/routes/routes.component.js:121]: Once https://github.com/MetaMask/metamask-extension/pull/13097 is merged, we can slip themes from `Settings > Experimental`

Comment (GuillaumeRx) [reply ui/pages/routes/routes.component.js]: rebased to include #13097

**Comment by GuillaumeRx** [comment]
Waiting for https://github.com/MetaMask/design-tokens/pull/46

**Comment by darkwing** [comment]
This is looking pretty good; the only thing I'm finding a bit hard to read are the network and account dropdown:
Network
NetworksDropdown
MenuTough

**Comment by darkwing** [comment]
Possibly `overlay.alternative` might be a better option?

File: ui/css/itcss/components/network.scss [hunk 2]
```diff
@@ -138,15 +138,15 @@
...
+  background-color: var(--color-border-default);
 }
 
 .network-dropdown-title {
   @include H4;
 
   height: 25px;
   width: 120px;
-  color: var(--white);
+  color: var(--color-background-default);
```
Comment (georgewrmarshall) [inline ui/css/itcss/components/network.scss:149]: I think you may want to use overlay.inverse here if it's text on top of the component that uses overlay.default or overlay.alternative

```suggestion
  color: var(--color-overlay-inverse);
```

File: ui/css/itcss/components/network.scss [hunk 3]
```diff
@@ -155,15 +155,15 @@
...
 
   &--link {
-    color: var(--white);
+    color: var(--color-background-default);
     cursor: pointer;
     text-decoration: underline;
 
     &:hover {
-      color: var(--white);
+      color: var(--color-background-default);
```
Comment (georgewrmarshall) [inline ui/css/itcss/components/network.scss:166]: Same here use `var(--color-overlay-inverse)`

**Review by georgewrmarshall — COMMENTED** [review]
Absolutely fantastic work 💯 a few tiny suggestions otherwise 🔥🔥🔥

**Review by darkwing — CHANGES_REQUESTED** [review]
Wow, this looks amazing!  The only issue I noticed is that some of the text colors are hard to read in dropdowns, and in one case the dropdown looks too light:

https://user-images.githubusercontent.com/46655/157913740-0fc250be-caa0-410d-9baa-1d2b7d6c1738.mp4

File: app/phishing.html [hunk 4]
```diff
@@ -1,67 +1,147 @@
...
-      .content__body p           { margin: 2em; }
-      .content__body p a         { text-decoration: underline; color: inherit; cursor: pointer; }
+      * {
+        margin: 0;
+        padding: 0;
+        box-sizing: border-box;
+      }
+      body,
+      html {
+        background-color: var(--color-secondary-alternative);
```
Comment (georgewrmarshall) [inline app/phishing.html:42]:
```suggestion
        background-color: var(--color-error-default);
```

**Review by georgewrmarshall — COMMENTED** [review]
Not sure if you saw my comment here. Is there some way to make those updates and then rebase from this branch 😓 https://github.com/MetaMask/metamask-extension/pull/13916#pullrequestreview-910763870

**Comment by darkwing** [comment]
I've addressed @georgewrmarshall 's feedback from the other PR on my `general-background-dark-mode` branch; let me know if you want me to push it to this.

**Review by darkwing — APPROVED** [review]

**Review by georgewrmarshall — APPROVED** [review]
ABSOLUTE 🔥 nice work!! 🚢 it

**Pull request merged**